test/extended/syncer/syncer.go

```diff
@@ -0,0 +1,76 @@
+package syncer
+
+import (
+	"os"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/kcp-dev/kcp-tests/test/extended/util"
+)
+
+var _ = g.Describe("[area/transparent-multi-cluster]", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		k = exutil.NewCLIWithWorkSpace("kcp-syncer")
+	)
+
+	g.It("Author:pewang-Critical-[Smoke][BYO] Validate creating, modifying and deleting a deployment from KCP get synced to the pcluster", func() {
+		pclusterKubeconfig := os.Getenv("PCLUSTER_KUBECONFIG")
+		if pclusterKubeconfig == "" {
+			g.Skip("No pcluster kubeconfig set for the test scenario")
+		}
+		k.SetupWorkSpaceWithNamespace()
+		myWs := k.WorkSpace()
+		k.SetGuestKubeconf(pclusterKubeconfig)
```

Comment:
I'm not sure if the word `Guest` as var name is proper. AFAIK, `guest` cluster is used in hypershift envs. In kcp envs, maybe it is quite proper? Maybe `PCluster` is better because `kubectl kcp workload sync -h` also uses `pcluster`?

Reply:
Hmm, good point. I haven't change it as pcluster as the client.go has many methods need to rename or refactor for current kcp. I think we could do it later. I could also change it now if needed.

Reply:
Addressed.